instrumentation/README.md

```diff
@@ -0,0 +1,11 @@
+# linux-autoinstrumentation
+
+This directory contains functionality to auto instrument Java programs that run on a Linux host.
```

Comment:
Will this need to support both amd64 and arm64?

Reply:
Are we packaging this only in the deb and rpm packages, or in the linux docker images as well?

Reply:
Haven't tested this in an arm environment, but my guess is that this will just work. However, my preference is that this PR just be for amd64 and we separate out arm support/testing.

I don't think this will work in a docker environment, so just deb/rpm. For security reasons, I think we should go with separate packages for this functionality.

cc: @aunshc